harvard-lts/etd-integration-tests
# Integration tests for various ETD types

**Integration tests for ETDs containing opaque, document, image, audio, text*
* * *

**JIRA Ticket**: https://at-harvard.atlassian.net/browse/ETD-307

# How should this be tested?

Tested in dev already.  To replicate:
Call the four tests and wait for load reports to appear in the  dts@hu.onmicrosoft.com group
https://ltsds-cloud-dev-1.lib.harvard.edu:10610/etd_with_images_test
https://ltsds-cloud-dev-1.lib.harvard.edu:10610/etd_with_opaque_test
https://ltsds-cloud-dev-1.lib.harvard.edu:10610/etd_with_audio_test
https://ltsds-cloud-dev-1.lib.harvard.edu:10610/etd_basic_test


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? yes

# Interested parties
@awoods
